pyproject.toml

```
@@ -179,7 +179,17 @@ markers = [
   "uses_concat_where",
   "cpu_only",
   "infinite_concat_where",
-  "gtfn_too_slow"
+  "gtfn_too_slow",
+  "to_performance_dashboard"
+]
+pythonpath = [
+  "model/atmosphere/diffusion",
+  "model/atmosphere/dycore",
+  "model/atmosphere/subgrid_scale_physics/microphysics",
+  "model/atmosphere/subgrid_scale_physics/muphys",
+  "model/common",
+  "model/driver",
+  "model/testing"
```

Tools is missing
```suggestion
  "model/testing",
  "tools/"
```
Just adding it here will break. `tools` package needs to be cleaned up:

`tools/tests/py2fgen/conftest.py` ->  `tools/tests/py2fgen/fixtures.py`
`tools/tests/conftest.py` ->  `tools/tests/py2fgen/conftest.py` (and maybe add `from fixtures import *` for now, if you don't want to add the `fixtures` import for all tests files)

---

I added `tools`, but didn't touch the other things. Seemed to work locally.